mypy.ini

```diff
@@ -21,16 +22,14 @@ exclude = (?x)(
 # *.extern modules that actually live in *._vendor will also cause attr-defined issues on import
 disable_error_code = attr-defined
 
-# - Avoid raising issues when importing from "extern" modules, as those are added to path dynamically.
-#   https://github.com/pypa/setuptools/pull/3979#discussion_r1367968993
+[mypy-pkg_resources._vendor.*,setuptools._vendor.*,setuptools.config._validate_pyproject.*,setuptools.tests.*]
+# Even when excluded there might be problems: python/mypy/issues/11936#issuecomment-1466764006
+# - Avoid raising issues when importing from "_vendor" modules
```

Comment:
`_validate_pyproject` is technically vendored but not in `_vendor`. This also generalises the comment if there's other vendored module locations to take into account (like `_distutils`, or something else in the future) 
```suggestion
# - Avoid raising issues when importing from vendored modules
```